test/Microsoft.ApplicationInsights.AspNet.Tests/Extensions/HttpRequestExtensionsTest.cs

```diff
@@ -2,7 +2,7 @@
 {
     using Microsoft.ApplicationInsights.AspNet.Extensions;
     using Microsoft.AspNet.Http;
-    using Microsoft.AspNet.Http.Core;
+    using Microsoft.AspNet.Http.Internal;
```

Comment:
What are we using from the `Http.Internal` namespace? Is it safe for us to continue doing so?